logger/consts/errors.py

```diff
@@ -0,0 +1 @@
+# TODO define constant error messages
```

Comment:
This can be a good idea. This can make unit testing to easier, by comparing/catching the error.

I suggest start doing this.